src/components/accessories/admin/operations/Operations.tsx

```diff
@@ -1,31 +1,52 @@
 import React, { useEffect } from "react";
 import classes from "./Operations.module.scss";
 import { useDispatch } from "react-redux";
+import {
+  deleteOperation,
+  getOperations,
+} from "../../../../state/operations/actions";
 import { OperationDTO } from "../../../../generated";
 import OperationTable from "./operationTable";
-import { getOperations } from "../../../../state/operations/actions";
-import { getOperationTypes } from "../../../../state/operationTypes/actions";
+import Button from "../../button/Button";
+import { useNavigate } from "react-router";
+import { useTranslation } from "react-i18next";
+import { PATHS } from "../../../../consts";
 
 export const Operations = () => {
+  const navigate = useNavigate();
   const dispatch = useDispatch();
+  const { t } = useTranslation();
 
   useEffect(() => {
     dispatch(getOperations());
-    dispatch(getOperationTypes());
   }, [dispatch]);
 
   const handleEdit = (row: OperationDTO) => {
-    // TODO
+    navigate(PATHS.operations_edit.replace("#id", row.code!), { state: row });

```

Comment:
maybe `PATHS.operations_edit` (and similar admin-relared paths) should be renamed `PATHS.admin_operations_edit` wdyt?